src/authorizer.js

```diff
@@ -41,6 +42,60 @@ class Authorizer {
     throw new Error('Unauthorized');
   }
 
+  async getApiKey(apiGateway, clientId) {
+    try {
+      const apiKeys = await apiGateway.getApiKeys({ nameQuery: clientId, includeValues: true, limit: 1 }).promise();
+      const apiKey = apiKeys.items[0];
+
+      if (apiKey && apiKey.id) {
+        return apiKey;
+      }
+      console.log(apiKeys);
```

Comment:
Maybe a descriptive message with this object? It would be easier to navigate the logs.